draft-ietf-masque-h3-datagram.md

```diff
@@ -182,102 +220,49 @@ all cases, the maximum permitted value of the H3_DATAGRAM SETTINGS parameter is
 1.
 
 
-# Datagram-Flow-Id Header Field Definition {#header}
+# RELIABLE_DATAGRAM HTTP/3 Frame Definition {#reliable-datagram-frame}
```

Comment:
I see why this frame is necessary for #38, since that was used to register flows, but I'd prefer to see this removed from this version of the proposal for simplicity and clarity. If people need it, it can be an extension later.